### PR TITLE
update string --> text

### PR DESCRIPTION
**Pull request by taylorfturner**
simply chaning from `string` and `random_string` to `text` and `random_text` for naming consistency between generators and the data profiler column profile (i.e. `text_column_profile`)

### CODE REVIEW

File: synthetic_data/distinct_generators/text_generator.py
Comment (taylorfturner) [inline synthetic_data/distinct_generators/text_generator.py:1]: rename whole file

File: synthetic_data/dataset_generator.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 from synthetic_data.distinct_generators.datetime_generator import random_datetimes
 from synthetic_data.distinct_generators.float_generator import random_floats
 from synthetic_data.distinct_generators.int_generator import random_integers
-from synthetic_data.distinct_generators.string_generator import random_string
+from synthetic_data.distinct_generators.text_generator import random_text
```
Comment (taylorfturner) [inline synthetic_data/dataset_generator.py:15]: fix imports

File: synthetic_data/dataset_generator.py [hunk 2]
```diff
@@ -60,7 +60,7 @@ def generate_dataset(
         "float": random_floats,
         "categorical": random_categorical,
         "datetime": random_datetimes,
-        "string": random_string,
+        "text": random_text,
```
Comment (taylorfturner) [inline synthetic_data/dataset_generator.py:63]: fix mapping

File: tests/distinct_generators/test_string_generator.py
Comment (taylorfturner) [inline tests/distinct_generators/test_string_generator.py:1]: rename whole file to `test_text_generator.py` below

File: tests/distinct_generators/test_text_generator.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import unittest
+
+import numpy as np
+import pandas as pd
+
+from synthetic_data.distinct_generators.text_generator import random_text
+
+
+class TestTextGeneratorFunctions(unittest.TestCase):
```
Comment (taylorfturner) [inline tests/distinct_generators/test_text_generator.py:9]: rename class name

File: tests/distinct_generators/test_text_generator.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import unittest
+
+import numpy as np
+import pandas as pd
+
+from synthetic_data.distinct_generators.text_generator import random_text
```
Comment (taylorfturner) [inline tests/distinct_generators/test_text_generator.py:6]: rename import

File: tests/distinct_generators/test_text_generator.py [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+
+class TestTextGeneratorFunctions(unittest.TestCase):
+    def setUp(self):
+        self.rng = np.random.default_rng(12345)
+
+    def test_return_type(self):
+        text_arr = random_text(self.rng)
```
Comment (taylorfturner) [inline tests/distinct_generators/test_text_generator.py:14]: variable array naming from `str_arr` to `text_arr`

File: tests/test_dataset_generator.py [hunk 6]
```diff
@@ -22,8 +22,8 @@ def setUp(self):
                 "end_date": pd.Timestamp(2022, 12, 22),
             },
             {
-                "generator": "string",
-                "name": "str",
+                "generator": "text",
+                "name": "txt",
```
Comment (taylorfturner) [inline tests/test_dataset_generator.py:26]: rename for consistency

File: tests/test_dataset_generator.py [hunk 7]
```diff
@@ -89,7 +89,7 @@ def test_generate_custom_dataset(self):
             ),
         ]
         expected_df = pd.DataFrame.from_dict(
-            dict(zip(["int", "dat", "str", "cat", "flo"], expected_data))
+            dict(zip(["int", "dat", "txt", "cat", "flo"], expected_data))
```
Comment (taylorfturner) [inline tests/test_dataset_generator.py:92]: rename so text passes

**Review by JGSweets — APPROVED** [review]

**Review by tazitoo — APPROVED** [review]
LGTM

**Review by danielbarcklow — APPROVED** [review]

**Pull request merged**